src/Symfony/Component/Workflow/Tests/EventListener/GuardListenerTest.php

```diff
@@ -69,6 +69,18 @@ public function testWithSupportedEventAndAccept()
         $this->assertTrue($event->isBlocked());
     }
 
+    /**
+     * @expectedException \Symfony\Component\Workflow\Exception\InvalidTokenConfigurationException
+     * @expectedExceptionMessage There are no token available for workflow unnamed
+     */
+    public function testWithNoTokenStorage()
```

Comment:
actually, there is a token storage but no token

Reply:
Updated!